src/main/java/org/scalameter/japi/annotation/scopes.java

```diff
@@ -0,0 +1,33 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Allows to set custom contexts for scopes.
```

Comment:
I'd do that yes - generally, it's better to use the present participle in these cases instead of infinitive.